core/src/apps/cardano/sign_tx.py

```diff
@@ -724,11 +725,11 @@ def _should_hide_output(output: List[int], inputs: List[CardanoTxInputType]) ->
     for tx_input in inputs:
         inp = tx_input.address_n
         if (
-            len(output) != BIP_PATH_LENGTH
+            len(output) != ADDRESS_PATH_INDEX + 1
```

Comment:
this is somewhat less readable. I'd rather keep the code as it was, except for the bugfix